src/hooks/screen/useChatScreenHook.ts

```diff
@@ -3,12 +3,14 @@ import 'react-native-get-random-values';
 
 import * as React from 'react';
 import {useMutation} from 'react-query';
+import {useRecoilState} from 'recoil';
 import {useRoute} from '@react-navigation/core';
 import {v4 as uuid} from 'uuid';
 
 import ChannelList from '../../database/schema/ChannelListSchema';
 import ChatSchema from '../../database/schema/ChatSchema';
 import UserSchema from '../../database/schema/UserSchema';
+import currentChatScreenAtom from '../../atom/currentChatScreenAtom';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import order seems to be inconsistent. It's a good practice to keep the imports organized, usually with library imports at the top, followed by absolute imports and then relative imports. This helps in maintaining readability and understanding dependencies better.
```diff
- import * as React from 'react';
- import {useMutation} from 'react-query';
- import {useRecoilState} from 'recoil';
- import {useRoute} from '@react-navigation/core';
- import {v4 as uuid} from 'uuid';
- import ChannelList from '../../database/schema/ChannelListSchema';
- import ChatSchema from '../../database/schema/ChatSchema';
- import UserSchema from '../../database/schema/UserSchema';
- import currentChatScreenAtom from '../../atom/currentChatScreenAtom';
- import useAnalyticUtilsHook from '../../libraries/analytics/useAnalyticUtilsHook';
- import useChatUtilsHook from '../core/chat/useChatUtilsHook';
- import useDatabaseQueueHook from '../core/queue/useDatabaseQueueHook';
+ import * as React from 'react';
+ import {useMutation} from 'react-query';
+ import {useRecoilState} from 'recoil';
+ import {useRoute} from '@react-navigation/core';
+ import {v4 as uuid} from 'uuid';
+ import currentChatScreenAtom from '../../atom/currentChatScreenAtom';
+ import ChannelList from '../../database/schema/ChannelListSchema';
+ import ChatSchema from '../../database/schema/ChatSchema';
+ import UserSchema from '../../database/schema/UserSchema';
+ import useAnalyticUtilsHook from '../../libraries/analytics/useAnalyticUtilsHook';
+ import useChatUtilsHook from '../core/chat/useChatUtilsHook';
+ import useDatabaseQueueHook from '../core/queue/useDatabaseQueueHook';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->